app/src/main/java/com/tangem/tap/features/tokens/impl/presentation/TokensListFragment.kt

```diff
@@ -40,6 +41,10 @@ internal class TokensListFragment : Fragment() {
                 }
 
                 TangemTheme {
+                    val statusBarColor = TangemTheme.colors.background.secondary
+                    SystemBarsEffect {
+                        setStatusBarColor(color = statusBarColor)
```

Comment:
Лучше сразу и для navigation bar выставить
```suggestion
                        setSystemBarsColor(color = statusBarColor)
```